wlib/LoginDlg.cpp

```diff
@@ -699,6 +730,24 @@ class CLoginDlgAMT : public CDialogImpl<CLoginDlgAMT>,
 		return 0;
 	}
 
+	LRESULT OnBnClickedChangePassword(WORD /*wNotifyCode*/, WORD /*wID*/, HWND /*hWndCtl*/, BOOL& /*bHandled*/)
+	{
+		DoDataExchange(true);
+		GetIConfig(QUERYBUILDER_CFG)->Set(GLOBAL_USER, m_UserLHS);
+		GetIConfig(QUERYBUILDER_CFG)->Set(GLOBAL_SERVER_ACCOUNT, m_configLHS->Get(GLOBAL_SERVER_ATTRIBUTE));
+		DoModalChangePassword();
+		return 0;
+	}
+
+	LRESULT OnBnClickedChangePasswordRHS(WORD /*wNotifyCode*/, WORD /*wID*/, HWND /*hWndCtl*/, BOOL& /*bHandled*/)
+	{
+		DoDataExchange(true);
+		GetIConfig(QUERYBUILDER_CFG)->Set(GLOBAL_USER, m_UserRHS);
+		GetIConfig(QUERYBUILDER_CFG)->Set(GLOBAL_SERVER_ACCOUNT, m_configRHS->Get(GLOBAL_SERVER_ATTRIBUTE));
+		DoModalChangePassword();
```

Comment:
Has this been tested?

Reply:
Half way. Yes, I need to test this. I was first wanting a general feel from you before finishing.